pybop/pipelines/_pybamm_pipeline.py

```diff
@@ -256,94 +234,75 @@ def build(self) -> None:
         self._built_model = model
         self._solver = self._solver.copy()  # reset solver for new model
 
-    def solve(
-        self, calculate_sensitivities: bool = False
-    ) -> list[pybamm.Solution | FailedSolution]:
-        """
-        Run the simulation using the built model and solver.
+    def _solve_in_time_without_rebuild(
+        self, inputs: list[Inputs]
+    ) -> list[pybamm.Solution]:
+        """Solve in time without rebuilding the PyBaMM model."""
+        if len(inputs) == 1:
```

Comment:
do you need this, can you just pass in the list of length 1? I would hope if we (pybamm) did things correctly this should give the same results at the same speed :)